src/esm_runscripts/filedicts.py

```diff
@@ -309,7 +310,7 @@ def _resolve_paths(self) -> None:
         """
         self.locations = {
             "work": pathlib.Path(self._config["general"]["thisrun_work_dir"]),
-            "computer": pathlib.Path(self["path_in_computer"]),
+            "computer": self.path_in_computer, # Already Path type from _init_
```

Comment:
I would leave it like it is. Conversion of `pathlib.Path` to `Path` again is a zero-operation, and then it is consistent with the other ones, and you save yourself a comment 😉

Reply:
It breaks if `self["path_in_computer"]` is `None` and that's the problem.

Reply:
That's annoying.

Reply:
yeah, I wish there was a PathNone or something like that...

Reply:
I have a good solution for that. Check for my `FileTypes` enumeration

Reply:
Looks good so far. I can refactor your `path` `None` things with my `FileTypes` later on.